src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/Registry.cs

```diff
@@ -64,7 +64,7 @@ private static RegistryKey GetBaseKeyFromKeyName(string keyName, out string subK
                 return baseKey;
             }
 
-            throw new ArgumentException(SR.Format(SR.Arg_RegInvalidKeyName, nameof(keyName)), nameof(keyName));
+            throw new ArgumentException(SR.Arg_RegInvalidKeyName, nameof(keyName));
```

Comment:
We can change resource strings if we want. In some cases, perhaps including more info might be worthwhile. Eg.
```c#
throw new ArgumentException(SR.Format(SR.Arg_RegInvalidKeyName, keyName), nameof(keyName));
```
with maybe
```
Registry key name "{0}" must start with a valid base key name.
```
up to you.